layout/js/animate.js

```diff
@@ -109,6 +114,12 @@ function hovertext(text, evt){
     tooltip_bg.setAttribute("class","tooltip-box");
     tool_pt.setAttribute("class","tooltip-box");
     tooltip_bg.setAttribute("width", length+12);
+    if(hoverTimer){
+      clearTimeout(hoverTimer);
+    }
+    hoverTimer = setTimeout(function(){
+      ga("send", "event", "figure", evt.target.id + ":" + text);
```

Comment:
how about we just use the `evt.target.id`?

would need to add `                  id = sprintf('inactive-%s', shown.inactive.gages$site_no[i]), ` to line 172 of visualize